src/function.ts

```diff
@@ -418,3 +418,10 @@ export const increment = (n: number): number => {
 export const decrement = (n: number): number => {
   return n - 1
 }
+
+/**
+ * @since 1.18.0
+ */
+export function absurd<A>(_: never): A {
+  throw new Error('absurd')
```

Comment:
Could put a more descriptive error here. "Called `absurd` function which should be uncallable".